smoke/IotEdgeQuickstart/Program.cs

```diff
@@ -92,6 +93,9 @@ class Program
         [Option("--leave-running=<All/Core/None>", CommandOptionType.SingleOrNoValue, Description = "Leave IoT Edge running when the app is finished")]
         public LeaveRunning LeaveRunning { get; } = LeaveRunning.None;
 
+        [Option("--no-deployment", CommandOptionType.NoValue, Description = "Doesn't start IoT Edge module deployment")]
```

Comment:
For consistency with other descriptions, should be `Don't deploy Edge Hub and temperature sensor modules`.